src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -1191,6 +1229,8 @@ def heat(scenario):
             "central_gas_boiler": read_costs(
                 costs, "central gas boiler", "VOM"
             ),
+            "rural_biomass_boiler": 51,  # considering an efficiency of 0.8
```

Comment:
A source for the actual costs would be very helpful

Reply:
Addressed in 7c07862fa3d78a6bc6834faaca34207b3e0f36bb